### ISSUE TITLE
Is this lib still maintained?

### ISSUE THREAD

**Issue by idish** [afollestad/material-camera#189]
I'm not sure if I should rely on this lib in my app as I see there are multiple critical opened issues and some PRs that are not yet merged nor commented by the author. 
It is really a shame as I think this library is so much usable for so many people.

**Comment by tylerkrupicka** [comment]
@idish I'll leave a comment here in case others look here for insights in to the current state with regards to @sugarmanz and I. I'm one of a few collaborators on the project although I was only added in the last ~8 months or so. @sugarmanz and I added the stillshot feature and a number of bug fixes (with some help from other contributors) late last year while using this repository for a personal project. Between @sugarmanz and I we have a small collection of devices spanning most android versions and manufacturers. Part of the difficulty of maintaining this library is that any change could break devices different from the one you test on (_cough cough Samsung_) , so we spent a lot of time making sure we had verified everything. 

If you want an example of the PRs we submitted take a look at https://github.com/afollestad/material-camera/pull/130. If you have a feature you are working on, some documentation and testing would be really helpful.

We're both college students and have been out on an internship this spring so we don't have a lot of the devices with us and haven't been as actively working on our project. We'll be headed back soon so that may change based on our schedule.

As far as what everyone else can do to help us keep the project moving forward.. please help us test some of the PRs and other things. I can take a look at the current PRs once we head back to school but they're out of date with master and I'm going to have to spend hours building the app for each of my devices for QA testing. 

Feel free to post any questions here and I can try to jump in and answer them. Additionally, if you have something you'd like us to help with tag us in the ticket and we can jump in.

TLDR: Might be some time to do development while back at school, please help out by documenting and testing PRs. Thanks!
